src/SFA.DAS.AssessorService.Web.UnitTests/StandardControllerTests/OptInStandardTests.cs

```diff
@@ -178,15 +179,16 @@ public void OptInStandardVersion_ReturnsArgumentException_WhenPostCalledWithNull
         }
 
         [Test]
-        public void OptInStandardVersion_ReturnsArgumentException_WhenPostCalledWithExistingVersionForStandard()
+        public async Task OptInStandardVersion_ReturnsArgumentException_WhenPostCalledWithExistingVersionForStandard()
```

Comment:
This should be a redirect test of the view model being invalid

Reply:
Changed